specification/network/resource-manager/Microsoft.Network/stable/2021-03-01/virtualWan.json

```diff
@@ -7912,6 +8146,98 @@
         }
       },
       "description": "Office365 breakout categories."
+    },
+    "ListRoutingIntentResult": {
+      "description": "List of the routing intent result and a URL nextLink to get the next set of results.",
+      "type": "array",
```

Comment:
change array type to object.  If it caused the model validation failures, then please fix the examples.  
add below to body section in example:
```code
"value": []
```